csFastFloat/FastFloatParser.cs

```diff
@@ -676,7 +659,7 @@ unsafe static internal ParsedNumberString ParseNumberString(byte* p, byte* pend,
         {
           return answer;
         }
-        if (!Utils.is_integer(*p, out uint digit) && (*p != decimal_separator)) // culture info ?
+        if (!Utils.is_integer(*p, out uint _) && (*p != decimal_separator)) // culture info ?
```

Comment:
```suggestion
        if (!Utils.is_integer(*p, out _) && (*p != decimal_separator)) // culture info ?
```
It's a nit but this is proper discard syntax.

Reply:
Thanks! Updated e8ddb0f665d9a5521325428c425de288b176e5b4